### PR TITLE
added dbhandler methods to fetch data from db_handler file and import…

### PR DESCRIPTION
**Pull request by Spillahm**
… statements to import db_handler and data_presenter to main.py

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -59,6 +61,26 @@ def get_city():
...
+db_handler.add_things_to_do(things_to_do_data)
+db_handler.add_geo_location(geo_location_data)
+
+# finding geo location data
+place_name = "place name"
+geo_location = DataPresenter.get_geolocation_data(place_name)
+print(geo_location)
+
 
 if __name__ == "__main__":
```
Comment (claraj) [inline main.py:82]: There are two checks `if __name__ == "__main__"` do you need both?

File: main.py [hunk 2]
```diff
@@ -59,6 +61,26 @@ def get_city():
...
     return city.capitalize()
 
+def store_data():
+    db_handler = DBHandler('records_db.sqlite')
+
+    db_handler.create_table()# creates table if it does not exist
+
+#the following lines add data using dbhandler methods
+
+db_handler.add_travel_advisory(advisory_data)
```
Comment (claraj) [inline main.py:71]: Should these lines be in a function? Where are the advisory_data, food_data, things_to_do_data and geo_location_data defined?

Comment (Spillahm) [reply main.py:71]: They are defined in db_handler and data-presenter

File: main.py (hunk 1 above)
Comment (Spillahm) [reply main.py:82]: I am only using the last one to store data I am not sure who had added the other one there

Comment (sndmnska) [reply main.py:82]: Which is better?  I forget how exactly this check worked.    If we leave yours for storing data, does that mean the rest of it doesn't work? Actual question, I'm not sure how this works, sorry.

**Review by sndmnska — APPROVED** [review]
Admittedly, I'm short on time, I think it's probably good.  Nothing's jumping out at me as wrong at least.

File: main.py [hunk 3]
```diff
@@ -59,6 +61,26 @@ def get_city():
...
+db_handler.addd_food_to_eat(food_data)
```
Comment (sndmnska) [inline main.py:72]: I noticed this was mispelled.  I'll make note of it to fix this later as needed.

**Pull request merged**